bin.js

```diff
@@ -13,7 +13,9 @@ if (wantedPM !== 'npm' && wantedPM !== 'cnpm' && wantedPM !== 'pnpm' && wantedPM
   process.exit(1)
 }
 const usedPM = whichPMRuns()
-if (usedPM && usedPM.name !== wantedPM) {
+const isInstallAsDependency = process.env.INIT_CWD.indexOf('node_modules') > -1
+console.log(usedPM, isInstallAsDependency, process.env.INIT_CWD)
```

Comment:
```suggestion
const cwd = process.env.INIT_CWD || process.cwd()
const isInstallAsDependency = cwd.includes('node_modules')
```
**Suggested**: Remove debug console.log and fallback to `process.cwd()` when there is no `INIT_CWD`.